templates/c2s-gp-finder/0/rancher-compose.yml

```diff
@@ -11,6 +11,12 @@ catalog:
       required: true
       default: "latest"
       type: "string"
+    - variable: "gp_finder_db_docker_image_tag"
```

Comment:
The variable will need to be `profiles_db_docker_image_tag` for it to work with the CI scripts. @neilbmclaughlin can you confirm?
If it does, probably the description and label here will want to be updated along with where it is used in `docker-compose` i.e. https://github.com/nhsuk/nhsuk-rancher-templates/pull/28/files#diff-a5f45e5d628f0630d457041224a3c35eR23